content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,243 @@
 
-.. image:: usage/usage_05.png
-    :align: center
+Multiple items in a sales order (SO) can be separated into different packages to accommodate the
+products. For example, a :abbr:`SO (Sales Order)` that has 20 boxes of pencils and 4 boxes of
+erasers can be separated into two separate packages, each containing 10 boxes of pencils and 2 boxes
+of erasers.
 
-.. image:: usage/usage_06.png
-    :align: center
+.. note::
+   Products do *not* have to be divided equally. Products can be divided into as many packages that
+   are needed to accommodate the :abbr:`SO (Sales Order)`.
 
-Packaging
-=========
+In Odoo, the quantity of products in each package needs to be recorded, so there is a full history
+for each product, including which package each item is shipped out in.
 
-The packaging is the physical container that protects your product. If
-you are selling computers, the packaging contains the computer with the
-notice and the power plug.
+To use this method, ensure the :guilabel:`Packages` option is enabled, by navigating to
+:menuselection:`Inventory app --> Configuration --> Settings --> Operations`. Click the
+:guilabel:`Packages` check box, and then :guilabel:`Save` to activate the feature.
 
-In Odoo, packagings are used for indicative purposes on sale orders.
-They can be specified on the product form, in the inventory tab.
+On a delivery order, assign which package to use by clicking on the :guilabel:`≣ (Detailed
+Operations)` icon to the right of each product in the :guilabel:`Operations` tab.
 
-.. image:: usage/usage_07.png
-    :align: center
+.. image:: usage/detailed-operations-icon.png
+   :align: center
+   :alt: Find detailed operations icon to the right on the product line.
 
-.. image:: usage/usage_08.png
-    :align: center
+Clicking the :guilabel:`≣ (Detailed Operations)` icon of a product reveals a pop-up window. In this
+window, the :guilabel:`Product` name, :guilabel:`Demand`, and :guilabel:`Quantity Done` are clearly
+displayed above the customizable product line.
 
-.. note::
-        Another useful use of the packaging is for product reception. By
-        scanning the barcode of the packaging, Odoo adds the number of units
-        contained in the packing on the picking.
+Here, the warehouse location the product is coming from can be modified, in addition to the
+:guilabel:`Source Package` and :guilabel:`Destination Package`, which represents what package(s) the
+product(s) will be packaged into.
+
+Click :guilabel:`Add a line` to include each additional package used. Enter the desired amount in
+the :guilabel:`Done` field to specify the amount of products that will go into the
+:guilabel:`Destination Package`.
+
+Multiple packages in a single delivery
+--------------------------------------
+
+Although a :abbr:`DO (Delivery Order)` is typically associated with a single package, orders can be
+split by clicking the :guilabel:`≣ (Detailed Operations)` icon next to the products on a :abbr:`SO
```

Comment:
grammar (again)
double-check whole doc for this

yes, even though SO stands for "sales order" and starts with a consonant, it's _sound_ starts with a vowel, as in "Ess-Oh", versus DO which starts with a hard consonant sound (and would therefore use an "a" instead of an "an" beforehand)

```suggestion
split by clicking the :guilabel:`≣ (Detailed Operations)` icon next to the products on an :abbr:`SO
```